pandas/core/series.py

```diff
@@ -3000,8 +3000,9 @@ def sort_index(
             Axis to direct sorting. This can only be 0 for Series.
         level : int, optional
             If not None, sort on values in specified index level(s).
-        ascending : bool, default true
-            Sort ascending vs. descending.
+        ascending : bool or list of bools, default True
```

Comment:
if list of bools is allowed, should that show up in the annotation?

Reply:
See the discussion with WillAyd above; mypy doesn't seem to like that because `nargsort` gets called when we don't have a MultiIndex and it expects a single bool